src/app-layout/visual-refresh-toolbar/drawer/global-bottom-drawer.tsx

```diff
@@ -0,0 +1,309 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useEffect, useRef } from 'react';
+import { Transition } from 'react-transition-group';
+import clsx from 'clsx';
+
+import { InternalItemOrGroup } from '../../../button-group/interfaces';
+import ButtonGroup from '../../../button-group/internal';
+import PanelResizeHandle from '../../../internal/components/panel-resize-handle';
+import customCssProps from '../../../internal/generated/custom-css-properties';
+import { usePrevious } from '../../../internal/hooks/use-previous';
+import { getLimitedValue } from '../../../split-panel/utils/size-utils';
+import { Focusable } from '../../utils/use-focus-control';
+import { getDrawerStyles } from '../compute-layout';
+import { AppLayoutWidgetizedState, InternalDrawer } from '../interfaces';
+import { useResize } from './use-resize';
+
+import sharedStyles from '../../resize/styles.css.js';
+import testutilStyles from '../../test-classes/styles.css.js';
+import styles from './styles.css.js';
+
+export function AppLayoutBottomDrawerWrapper({ widgetizedState }: { widgetizedState: AppLayoutWidgetizedState }) {
+  const { activeGlobalBottomDrawerId, bottomDrawers } = widgetizedState;
+  const openBottomDrawersHistory = useRef<Set<string>>(new Set());
+  useEffect(() => {
+    if (activeGlobalBottomDrawerId) {
+      openBottomDrawersHistory.current.add(activeGlobalBottomDrawerId);
+    }
+  }, [activeGlobalBottomDrawerId]);
```

Comment:
Why is it happening inside useEffect? The `openDrawersHistory` with a similar use-case does not have an effect: https://github.com/cloudscape-design/components/blob/f724c4ec701b5e750ad74d19269c98f284019f95/src/app-layout/visual-refresh-toolbar/drawer/global-drawers.tsx#L31

1. If you fixed a bug, how about propagating the fix to the other place?
2. If it is not needed, then let's remove